### PR TITLE
fix(RBAC): fix scope for release tags

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: rbac/rbac.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func WithContext(ctx context.Context, scope *Scope) context.Context {
 func ExtractScopeFromHeader(r *http.Request) *Scope {
 	groupStr := r.Header.Get(ScopeHeader)
 	tagsStr := r.Header.Get(ScopeReleaseTagsHeader)
-	if len(groupStr) > 0 {
+	if len(groupStr) || len(tagsStr) > 0 {
```
Comment (merlin-northern) [inline rbac/rbac.go:52]: just to double check: are we sure that for else to this if we are returning nil?

**Review by merlin-northern — APPROVED** [review]
one double-check kiind of comment.

File: rbac/rbac.go (hunk 1 above)
Comment (kjaskiewiczz) [reply rbac/rbac.go:52]: sorry, but I don't understand the comment; can you rephrase?

Comment (kjaskiewiczz) [reply rbac/rbac.go]: that is something we were already doing (btw, I fixed the condition) and is something we don't want to change

Comment (merlin-northern) [reply rbac/rbac.go]: copy that

Comment (merlin-northern) [reply rbac/rbac.go]: I noticed the condition fix. and I was copying this code to the comment -- still failed to notice the new syntax ;-)
